src/stats/dendrogram.jl

```diff
@@ -65,19 +66,22 @@ function recursive_dendrogram_points(node, nodes, ret_points = Point2f[], ret_co
 
     recursive_dendrogram_points(child1, nodes, ret_points, ret_colors; branch_shape, groups)
     recursive_dendrogram_points(child2, nodes, ret_points, ret_colors; branch_shape, groups)
-    return ret_points, ret_colors
 end
 
 
 function Makie.plot!(plot::Dendrogram{<: Tuple{<: Dict{<: Integer, <: Union{DNode{2}, DNode{3}}}}})
     args = @extract plot (color, groups)
 
-    points_vec = Observable{Vector{GeometryBasics.Point{2, Float32}}}([[0,0]])
-    colors_vec = Observable{Any}([NaN])
+    points_vec = Observable{Vector{GeometryBasics.Point{2, Float32}}}([])
+    colors_vec = Observable{Any}([])
 
     length(plot[1][])>1 && lift(plot[1], plot.branch_shape, plot[:color]) do nodes, branch_shape, color
+        points_vec[] = []
+        colors_vec[] = []
```

Comment:
```suggestion
        points_vec.val = []
        colors_vec.val = []
        notify(points_vec)
        notify(colors_vec)
```

Otherwise this will throw an error, since the length of the color vector is not the same as the length of the point vector.